clusterloader2/pkg/measurement/common/probes/probes.go

```diff
@@ -228,46 +229,23 @@ func (p *probesMeasurement) checkProbesReady() (bool, error) {
 		p.framework.GetClientSets().GetClient(), isProbeTarget, expectedTargets)
 }
 
-func isProbeTarget(t prometheus.Target) bool {
-	return t.Labels["namespace"] == probesNamespace
-}
-
-func parseThresholds(params map[string]interface{}) (map[string]*measurementutil.LatencyMetric, error) {
-	thresholds := make(map[string]*measurementutil.LatencyMetric)
-	t, ok := params["thresholds"]
-	if !ok {
-		return thresholds, nil
-	}
-	for name, thresholdVal := range t.(map[string]interface{}) {
-		threshold, err := time.ParseDuration(thresholdVal.(string))
-		if err != nil {
-			return nil, err
-		}
-		thresholds[name] = makeLatencyThreshold(threshold)
+func (p *probesMeasurement) createSummary(latency measurementutil.LatencyMetric) (measurement.Summary, error) {
+	content, err := util.PrettyPrintJSON(&measurementutil.PerfData{
+		Version: p.config.MetricVersion,
+		// NOTE(oxddr): for backward compatibility we need to pass "Probes" to be used a metric label.
+		DataItems: []measurementutil.DataItem{latency.ToPerfData("Probes")},
+	})
+	if err != nil {
+		return nil, err
 	}
-	return thresholds, nil
+	return measurement.CreateSummary(p.config.MetricName, "json", content), nil
```

Comment:
Why does it have to be MetricName? It would be much more intuitive to make it prober.Name

Reply:
This is the same issue as above. Going to drop that.